ui/app/mainui/AppMain.qml

```diff
@@ -1625,6 +1626,7 @@ Item {
                 }
                 else
                     Global.openLink(link)
+                this.open = false
```

Comment:
Looks like `this.open = false` is eventually evaluated in all code branches. Maybe just call it in the beginning?

Reply:
Correct, I'll put it at the top